src/main/java/Duke.java

```diff
@@ -1,10 +1,92 @@
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Represents a Duke chatbot.
+ */
 public class Duke {
+
+    /**
+     * The Storage used for access/writing to files.
+     */
+    private Storage storage;
+    /**
+     * The filepath to read from/write to.
+     */
+    private String filePath;
```

Comment:
filePath does not change, maybe consider making it final? Applies for Ui as well